### PR TITLE
Cluster management

### PR DESCRIPTION
**Pull request by asvetlov**
Only smoke tests for getting lists are provided, I suspect that full provisioning is too heavy for running on every commit in automated mode.

### CODE REVIEW

File: neuromation/cli/admin.py [hunk 1]
```diff
@@ -26,4 +28,35 @@ def admin() -> None:
...
+    )
+
+
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("config", required=True, type=click.File(encoding="utf8", lazy=False))
+@async_cmd()
+async def add_cluster(root: Root, cluster_name: str, config: io.TextIOBase) -> None:
+    """
+    Print the list of available clusters.
```
Comment (dalazx) [inline neuromation/cli/admin.py]:
```suggestion
    Create a new cluster and start its provisioning
```

Comment (asvetlov) [reply neuromation/cli/admin.py]: Oooh, shame on me!

File: neuromation/api/admin.py [hunk 2]
```diff
@@ -38,8 +43,34 @@ def __init__(self, core: _Core, config: Config) -> None:
...
+                ret[cluster.name] = cluster
+            return ret
+
+    async def add_cluster(self, name: str, config: Dict[str, Any]) -> None:
+        url = self._config.admin_url / "clusters"
+        auth = await self._config._api_auth()
+        payload = {"name": name}
+        async with self._core.request("POST", url, auth=auth, json=payload) as resp:
+            resp
+        url = self._config.admin_url / "clusters" / name / "cloud_provider"
```
Comment (dalazx) [inline neuromation/api/admin.py]:
```suggestion
        url = self._config.api_url / "clusters" / name / "cloud_provider"
```

File: neuromation/cli/admin.py [hunk 3]
```diff
@@ -14,6 +15,35 @@ def admin() -> None:
...
+async def add_cluster(root: Root, cluster_name: str, config: IO[str]) -> None:
+    """
+    Create a new cluster and start its provisioning
+    """
+    if not root.quiet:
+        click.echo(f"Creating cluster {cluster_name}")
+    config_dict = yaml.safe_load(config)
+    await root.client._admin.add_cluster(cluster_name, config_dict)
+    if not root.quiet:
+        click.echo(f"Done")
```
Comment (tvoinarovskyi) [inline neuromation/cli/admin.py]: As currently, we don't poll cluster setup process maybe let us write something more verbose:
"Cluster successfully added and will be set up within 24 hours"

Comment (tvoinarovskyi) [reply neuromation/cli/admin.py]: And minor: not f-string

Comment (asvetlov) [reply neuromation/cli/admin.py]: The output is updated, thanks!

File: neuromation/cli/formatters/admin.py [hunk 4]
```diff
@@ -13,3 +15,12 @@ def __call__(self, clusters_users: Iterable[_ClusterUser]) -> List[str]:
             rows.append((user.user_name, user.role.value))
 
         return list(table(rows=rows))
+
+
+class ClustersFormatter:
+    def __call__(self, clusters: Iterable[_Cluster]) -> List[str]:
+        headers = [click.style("Name", bold=True)]
```
Comment (tvoinarovskyi) [inline neuromation/cli/formatters/admin.py]: Not a problem for 1 row, but if we add several we hit alignment issues. Created https://github.com/neuromation/platform-client-python/issues/1245 for that.

**Comment by tvoinarovskyi** [comment]
🚢 LGTM

**Pull request merged**